server/views/pages/continueReportSelect.njk

```diff
@@ -39,10 +36,9 @@
           "data-sort-value": report.displayName
         }
       },
-      { html: incidentDateHtml, attributes: {
+      { html: discoveryDateTimeHtml, attributes: {
           "data-sort-value": report.incidentDetails.dateTimeOfIncident
```

Comment:
The data sort value needs changing here - otherwise when the table is resorted by the user, it'll sort on the incident dateTIme and not the discoveryDateTime

Reply:
good spot!